README.asciidoc

```diff
@@ -0,0 +1,323 @@
+= %placeholder% 
+
+This is where we're going to write down what we've done so far.
```

Comment:
This should probably be removed or moved.

Reply:
Removed.